ruby-training/trainees/alan/test_katas/03/JadenCase/jaden_case.rb

```diff
@@ -0,0 +1,16 @@
+
+
+class JadenCase
+    
+    def self.to_array_of_words (sentence)
+        sentence.split()
+    end
+
+    def self.capitalize_words! (sentence)
```

Comment:
Please keep the parenthesis next to the method name with no spaces.
Also, the bang symbol `!` shouldn't be used in this case, as the method doesn't actually mutate the original sentence. What you're doing is returning a new array (map creates and returns a NEW array based on an existing collection).